OptimizelySDK.Tests/ProjectConfigTest.cs

```diff
@@ -863,5 +863,36 @@ public void TestGetAttributeIdWithInvalidAttributeKey()
             Assert.Null(Config.GetAttributeId("invalid_attribute"));
             LoggerMock.Verify(l => l.Log(LogLevel.ERROR, @"Attribute key ""invalid_attribute"" is not in datafile."));
         }
+
+        [Test]
+        public void TestCreateThrowsWithNullDatafile()
+        {
+            var exception = Assert.Throws<ConfigParseException>(() => ProjectConfig.Create(null, null, null));
+            Assert.AreEqual("Unable to parse null datafile.", exception.Message);
+        }
+
+        [Test]
+        public void TestCreateThrowsWithEmptyDatafile()
+        {
+            var exception = Assert.Throws<ConfigParseException>(() => ProjectConfig.Create("", null, null));
+            Assert.AreEqual("Unable to parse empty datafile.", exception.Message);
+        }
+
+        [Test]
+        public void TestCreateThrowsWithUnsupportedDatafileVersion()
+        {
+            var unsupportedConfig = JsonConvert.DeserializeObject<ProjectConfig>(TestData.Datafile);
```

Comment:
Please create a separate mock unsupported datafile and load it into this test case